### PR TITLE
Fix broken links in MegaMenu

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes

Fixes https://github.com/Qiskit/qiskit.org/issues/2595

## Implementation details

Updated links to pages that exist

## Note
I'm not sure how much of these links will need to be updated, once this PR merges https://github.com/Qiskit/platypus/pull/967. I just wanted to get the fix in, in case users come to the site and use the menu, between major merges. In other words, I didn't wanna leave broken links on the site for the sake of UX.

### CODE REVIEW

File: constants/megaMenuLinks.ts [hunk 1]
```diff
@@ -172,14 +172,14 @@ const GAMES_AND_DEMOS: MegaDropdownMenuGroup = {
...
-      url: `${baseUrl}${pathGamesAndDemos}/bonus-level-make-your-own-puzzles`,
+      url: `${baseUrl}${pathGamesAndDemos}/bonus-level-sandbox#bonus-148-0`,
       segment: {
         cta: 'bonus-level-make-your-own-puzzles', location: sectionGamesAndDemos
       }
     },
     {
       label: 'Level 5: Proving the Uniqueness of Quantum Variables',
-      url: `${baseUrl}${pathGamesAndDemos}/level-5-proving-the-uniqueness-of-quantum-variables`,
+      url: `${baseUrl}${pathGamesAndDemos}/bonus-level-sandbox#bonus-153-0`,
```
Comment (vabarbosa) [inline constants/megaMenuLinks.ts:182]: do we need these two sections? they are part of the same page as the item above (Bonus Level: Sandbox)?
in addition removing them will make it consistent with platypus:

**qiskit.org**
image

**platypus**
image

i think these two should be consistent (which i know should come once is https://github.com/Qiskit/web-components/issues/74 & https://github.com/Qiskit/platypus/issues/955 are addressed)

**Review by vabarbosa — CHANGES_REQUESTED** [review]
nice @techtolentino just one comment. thanks

File: constants/megaMenuLinks.ts (hunk 1 above)
Comment (techtolentino) [reply constants/megaMenuLinks.ts:182]: Good call... will fix now

Comment (techtolentino) [reply constants/megaMenuLinks.ts]: @vabarbosa - should be fixed and consistent w/ Textbook now...
Screen Shot 2022-05-12 at 4 13 29 PM

**Review by vabarbosa — APPROVED** [review]
@techtolentino looks good. thanks for the quick turnaround

**Pull request merged**